features/media-import.feature

```diff
@@ -60,6 +60,27 @@ Feature: Manage WordPress attachments
     And the {CACHE_DIR}/large-image.jpg file should exist
     And the return code should be 0
 
+  Scenario: Import a file as attachment from a local image and preserve the file modified time.
+    Given download:
+      | path                        | url                                              |
+      | {CACHE_DIR}/large-image.jpg | http://wp-cli.org/behat-data/large-image.jpg     |
+    And I run `touch -m 1505787257 {CACHE_DIR}/large-image.jpg`
+    And I run `wp option update gmt_offset -5`
+    And I run `wp media import {CACHE_DIR}/large-image.jpg --post_id=1 --filetime`
```

Comment:
This should be `When I run`, and include a `Then STDOUT should contain` statement